src/components/about-us/WhatWeDo.astro

```diff
@@ -2,7 +2,7 @@
 import { Image } from 'astro:assets';
 
 import Background from '@assets/about-us/background-what-we-do.png';
-import { getExercises } from '@data/exercises';
+import { getExercises } from '@data/real/exercises';
```

Comment:
```suggestion
import { getExercises } from '@data/exercises';
```

Couldn't we have a `@data/exercises` module that imports from real or test depending on the environment?